- name: right_arm
  target_to_source: { x: 0.0, y: 0.0, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 3.1415926 }
  target:
    begin:  torso_link
    end:    right_hand_link
  source: 
    parent:
    begin: right_shoulder
    end:   right_hand